<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminInformation</class>
 <widget class="QWidget" name="AdminInformation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>187</width>
    <height>164</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AdminInformation</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>images/myinformation.png</normaloff>images/myinformation.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="classLabel">
       <property name="text">
        <string>我的身份：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="classText">
       <property name="text">
        <string>identity</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="usernameLabel">
       <property name="text">
        <string>用户名：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="usernameText">
       <property name="text">
        <string>username</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="passwordLabel">
       <property name="text">
        <string>密码：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="passwordText">
       <property name="text">
        <string>password</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="pwdCheck">
       <property name="text">
        <string>显示密码</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="changeInformation">
       <property name="text">
        <string>修改信息</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="flushBtn">
       <property name="text">
        <string>刷新</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
